website/server/controllers/api-v3/groups.js

```diff
@@ -303,6 +305,18 @@ api.joinGroup = {
 
     let promises = [group.save(), user.save()];
 
+    if (inviter) {
+      inviter = await User.findById(inviter).select('notifications preferences.language').exec();
+
+      let data = {
+        headerText: common.i18n.t('invitationAcceptedHeader', inviter.preferences.language),
+        bodyText: common.i18n.t('invitationAcceptedBody', {groupName: group.name, username: user.auth.local.username},
+          inviter.preferences.language),
```

Comment:
I would put this back on the upper line and instead split the `{groupName: group.name, username: user.auth.local.username}` on multiple lines, like

```
data = {
  headerText: common.i18n.t('invitationAcceptedHeader', inviter.preferences.language),
  bodyText: common.i18n.t('invitationAcceptedBody', {
    groupName: group.name, 
    username: user.auth.local.username
  }, inviter.preferences.language),
}
```